### DISCUSSION TITLE
Disable Auto Update for Whisky?

### DISCUSSION THREAD

**Discussion by charlemagnec** [Whisky-App/Whisky#532] [Q&A] [answered]
I'm primarily using Whisky to play D2R, and with 2.1.0 it doesn't work at all. Problem is everytime I close the app, it automatically updates the next time - so I need to replace the App with an older copy every time. Is there a way to just disable automatic updating of the app?

**Comment by mavethee** [comment, marked as answer]
`Whisky` -> `Settings` -> `Updates` -> Uncheck both `Automatically Check for Updates`
Or download older version and once update is prompted or manually check for updates, uncheck box to update automatically.

As the game doesn't work, make sure to report an issue about it.

**Reply by IsaacMarovitz** [reply]
Worth noting that D2R is fixed in 2.1.3

**Reply by mavethee** [reply]
Ah, my bad, I don't own the game so missed the fix, sorry.
